README.md

```diff
@@ -0,0 +1,65 @@
+Roll 'm Up
+===
+
+This is Flatpak package for Roll 'm Up, classic pinball game. 
+published in Flathub.org: https://flathub.org/apps/details/nl.lostboys.rollemup. 
+
+Roll 'm Up is abandoned proprietary software created by Lostboys Media Lab 
+for Dommelsch Brewery from Netherlands in 1998.
+
+Known issues
+---
+
+- Highscore upload feature does not work: external servers are gone
+  for many years. You can no longer win beer in this game.
+- Resizing window sometimes causes the game to crash. 
+  Please, don't resize it. Content scaling doesn't work anyway.
+- It's impossible to use non-ASCII characters in highscore nickname.
+  Default "Player 1" nickname is fine.
+- Application always crashes when closing. We can live with that.
```

Comment:
It doesn't make sense to have it listed here. I'd move it to metainfo instead. No user will ever visit this README.

Reply:
Why? There is a link in README to this text.

Reply:
I'm telling you this from experience what users do. The app description should be self-sufficient and not depending on external resources.

Reply:
Okay. I kept link to the README for full list of known issue but also I added note about the most serious one: window resizing may crash the app. It seems to me other points from that list are not so important to show them in GNOME Software.